src/Components/Browser.JS/src/Boot.Server.ts

```diff
@@ -18,7 +31,9 @@ async function boot() {
     return loadEmbeddedResourcesAsync(bootConfig);
   });
 
-  const initialConnection = await initializeConnection(circuitHandlers);
+  // pass options.configureSignalR to configure the signalR.HubConnectionBuilder
+  const configureSignalR = (options && options.configureSignalR) || null;
```

Comment:
I don't think the parenthesis are required here.

Reply:
It still helps with clarity. I don't see a reason to take it out.